requirements/docs.in

```diff
@@ -6,6 +6,6 @@ myst-parser
 nbsphinx
 pydata-sphinx-theme>=0.13
 sphinx
-sphinx-autodoc-typehints
+sphinx-autodoc-typehints==1.23.0  # Higher versions require sphinx-7, which is not supported by sphinx-design yet
```

Comment:
Do we actually use sphinx-design in Scipp?

Reply:
This is not for Scipp. Is it needed for docs like Scitacean's?

Reply:
It is needed for tabs like here: https://scicatproject.github.io/scitacean/user-guide/installation.html But not for anything else in Scitacean.

And you added something similar in Markdown in this PR. I don't know if this is based on sphinx-design.